app/controllers/supervisor/comments_controller.rb

```diff
@@ -0,0 +1,19 @@
+class Supervisor::CommentsController < Supervisor::BaseController
+
+  def create
+    comment = Comment.new(comment_params)
+
+    if comment.save
```

Comment:
(review André) mail only if text is present. `after-save` on model?
(review Nathan) prefers the mailing to stay in the controller.